config/stylelint-config-carbon/package.json

```diff
@@ -24,22 +25,21 @@
     "provenance": true
   },
   "peerDependencies": {
-    "stylelint": "^15.0.0"
+    "stylelint": "^16.0.0"
   },
   "dependencies": {
-    "stylelint-a11y": "^1.2.3",
+    "@double-great/stylelint-a11y": "^3.0.2",
```

Comment:
I looked up the context for this for posterity: https://github.com/developer-stylechain/stylelint-a11y/issues/4#issuecomment-1927321522